src/common/workspace-store_test.ts

```diff
@@ -0,0 +1,123 @@
+import mockFs from "mock-fs"
+
+jest.mock("electron", () => {
+  return {
+    app: {
+      getVersion: () => '99.99.99',
+      getPath: () => 'tmp',
+      getLocale: () => 'en'
+    }
+  }
+})
+
+import { WorkspaceStore } from "./workspace-store"
+
+describe("workspace store tests", () => {
+  describe("for an empty config", () => {
+    beforeEach(async () => {
+      WorkspaceStore.resetInstance()
+      mockFs({ tmp: { 'lens-workspace-store.json': "{}" } })
+
+      await WorkspaceStore.getInstance<WorkspaceStore>().load();
+    })
+
+    afterEach(() => {
+      mockFs.restore()
+    })
+
+    it("default workspace should always exist", () => {
+      const us = WorkspaceStore.getInstance<WorkspaceStore>();
```

Comment:
Please change variable name in all test cases. It can be `store` or just `ws`, not the `us`.

Reply:
oops, good catch